apps/src/storage/dataBrowser/dataVisualizer/Snapshot.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import moment from 'moment';
+import BaseDialog from '@cdo/apps/templates/BaseDialog.jsx';
+import msg from '@cdo/locale';
+import DataVisualizer from './DataVisualizer';
+import * as dataStyles from '../dataStyles';
+
+class Snapshot extends React.Component {
+  static propTypes = {
+    records: PropTypes.array.isRequired,
+    numericColumns: PropTypes.arrayOf(PropTypes.string).isRequired,
+    chartType: PropTypes.number.isRequired,
+    bucketSize: PropTypes.string,
+    chartTitle: PropTypes.string,
+    selectedColumn1: PropTypes.string,
+    selectedColumn2: PropTypes.string,
+    filterColumn: PropTypes.string,
+    filterValue: PropTypes.string,
+    isMultiColumnChart: PropTypes.bool,
+    // Provided via Redux
+    tableName: PropTypes.string,
+    projectName: PropTypes.string
+  };
+
+  state = {
+    isSnapshotOpen: false
+  };
+
+  handleOpen = () => this.setState({isSnapshotOpen: true});
+
+  handleClose = () => this.setState({isSnapshotOpen: false});
+
+  copy = () => {}; // todo
+  save = () => {}; // todo
+
+  getDisplayableChartOptions = () => {
+    const options = [];
+    if (!!this.props.selectedColumn1) {
+      options.push(
+        `${
+          this.props.isMultiColumnChart
+            ? msg.dataVisualizerXValues()
+            : msg.dataVisualizerValues()
+        }: ${this.props.selectedColumn1}`
+      );
+    }
+    if (!!this.props.selectedColumn2) {
+      options.push(
+        `${msg.dataVisualizerYValues()}: ${this.props.selectedColumn2}`
+      );
+    }
+    if (!!this.props.bucketSize) {
+      options.push(
+        `${msg.dataVisualizerBucketSize()}: ${this.props.bucketSize}`
+      );
+    }
+    if (!!this.props.filterColumn && !!this.props.filterValue) {
+      options.push(
+        msg.dataVisualizerFilterDescription({
+          column: this.props.filterColumn,
+          value: this.props.filterValue
+        })
+      );
+    }
+    return options.join(', ');
+  };
+
+  render() {
+    return (
+      <div>
+        <button
```

Comment:
Should this be `<Button`?

Reply:
I don't think it needs to. In this case we don't need any of the extra stuff that the `Button` component has, so an html `<button>` seems fine.